internal/services/paloalto/next_generation_firewall_vhub_local_rulestack_resource.go

```diff
@@ -73,6 +75,10 @@ func (r NextGenerationFirewallVHubLocalRuleStackResource) Arguments() map[string
 
 		"destination_nat": schema.DestinationNATSchema(),
 
+		"plan_data": schema.PlanDataSchema(),
+
+		"marketplace_details": schema.MarketplaceDetailsSchema(),
```

Comment:
Suggest to arrange the newly added fields in alphabetical order. So please move `"marketplace_details": schema.MarketplaceDetailsSchema(),` to line 78. Thanks.